docs/docs/tutorial/index.md

```diff
@@ -314,10 +287,8 @@ cardano-cli query utxo --address $(cat credentials/bob-funds.addr) --out-file /d
 ```
 
 </TabItem>
-</Tabs>
```

Comment:
[CI](https://github.com/input-output-hk/hydra/actions/runs/9685477231/job/26725955291?pr=1469) fails because of this deletion.
Should revert this change to make it pass.

Reply:
There were a bunch of errors happening, and this was the only one getting reported. The main issue was that even after fixing this error, the same report continued to appear,, obscuring the actual underlying causes.